src/DataCell.js

```diff
@@ -194,10 +195,11 @@ export default class DataCell extends PureComponent {
     dataEditor,
     valueRenderer,
     valueViewer,
+    pressedKey,
   ) => {
     return (
       this.renderComponent(editing, cell) ||
-      this.renderEditor(editing, cell, row, col, dataEditor) ||
+      this.renderEditor(editing, cell, row, col, dataEditor, pressedKey) ||
```

Comment:
Are there any other event properties that it might make sense to have access to? I don't think we need to change this, but this is already a long-ish argument list, so I'm curious if you think it might get even bigger.

Reply:
@JustWalters Well, I think it might the last time I modify this repo at all as we are planning to get rid of 'react-datasheet' eventually. I could refactor it a bit and use an object but it would be better not to do that. The lib already pretty sensitive and buggy, especially considering that we use a version from this PR - https://github.com/nadbm/react-datasheet/pull/213 (see the author's reply to my comment in the PR), therefore I don't want to figure out what effect it might have. I tried to make as few changes as possible to implement the desired behavior.